client/me/profile-links/style.scss

```diff
@@ -1,3 +1,5 @@
+/** @format */
```

Comment:
I think the Prettier pragmas are no longer necessary after #29403 (also see #29404)